src/main/java/ladder/domain/Ladder.java

```diff
@@ -1,69 +1,76 @@
 package ladder.domain;
 
-import ladder.utils.ProbabilityStrategyRandom;
-
-import java.util.Comparator;
 import java.util.HashSet;
+import java.util.List;
 import java.util.Set;
 import java.util.stream.IntStream;
+import ladder.utils.ProbabilityStrategyRandom;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
 public class Ladder {
 
-    private final Column maxColumn;
-    private final Row maxRow;
-    private final Set<Line> lines;
+  private static final Logger log = LoggerFactory.getLogger(Ladder.class);
+  private final Column maxColumn;
+  private final Row maxRow;
+  private final Set<Line> lines;
+
+  public Ladder(Column maxColumn, Row maxRow, Set<Line> lines) {
```

Comment:
```suggestion
  private Ladder(Column maxColumn, Row maxRow, Set<Line> lines) {
```
정적 팩토리 메서드를 제공하고 있는데 public이 private으로 구현하는 것은 어떨까?

Reply:
넵 해당부분 반영하였습니다!